docs/contributing/building/building.md

```diff
@@ -142,11 +158,11 @@
     $ DEBUG=1 make
     ...
     GOOS=linux CC=clang GOARCH=amd64 CGO_CFLAGS="-I/home/gg/code/tracee/dist/libbpf" CGO_LDFLAGS="-lelf  -lz  /home/gg/code/tracee/dist/libbpf/libbpf.a" go build \
-	-tags core,ebpf \
-	-ldflags=" \
-		-extldflags \"\" \
-		-X main.version=\"v0.8.0-107-g121efeb\" \
-		" \
-	-v -o dist/tracee-ebpf \
-	./cmd/tracee-ebpf
+ -tags core,ebpf \
+ -ldflags=" \
+  -extldflags \"\" \
+  -X main.version=\"v0.8.0-107-g121efeb\" \
+  " \
+ -v -o dist/tracee-ebpf \
+ ./cmd/tracee-ebpf
```

Comment:
It seems broken:

![image](https://user-images.githubusercontent.com/3372117/222173562-d647020a-35cc-47b0-aee5-75845a7f8809.png)

Reply:
Hum. I think I know why.. when saving vscode is applying the linting automagically... hum, will fix it.